<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="button_pon">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>90</horstretch>
                <verstretch>90</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Пн</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_vtor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>90</horstretch>
                <verstretch>90</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Вт</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_sreda">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Ср</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_chetv">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Чт</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_pyatn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Пт</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_subb">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Сб</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_vsk">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Вс</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="Month">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>16</pointsize>
                <weight>9</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>month</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Day">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Day</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="button_week">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>24</pointsize>
              <weight>9</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Планы на жизнь</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_addTask">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 50px;
font: 75 30pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 0);
</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="task_1">
          <property name="styleSheet">
           <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-bottom: 0px;
</string>
          </property>
          <property name="text">
           <string>Task_1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_1">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
border:  2px solid grey;
border-top: 0px;</string>
          </property>
          <property name="text">
           <string>Time 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="task_2">
          <property name="styleSheet">
           <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-bottom: 0px;</string>
          </property>
          <property name="text">
           <string>Task_2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-top: 0px;</string>
          </property>
          <property name="text">
           <string>time_2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="task_3">
          <property name="styleSheet">
           <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-bottom: 0px;</string>
          </property>
          <property name="text">
           <string>task_3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_3">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-top: 0px;</string>
          </property>
          <property name="text">
           <string>time_3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="curr_task">
          <property name="styleSheet">
           <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-bottom: 0px;</string>
          </property>
          <property name="text">
           <string>Текущая задача</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_curr_task">
          <property name="styleSheet">
           <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-top: 0px;</string>
          </property>
          <property name="text">
           <string>Время</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="next_task">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-bottom: 0px;
</string>
            </property>
            <property name="text">
             <string>След задача</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="time_next_task">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid grey;
border-top: 0px;</string>
            </property>
            <property name="text">
             <string>время</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
